libs/vkd3d/command.c

```diff
@@ -6704,7 +6705,17 @@ static void STDMETHODCALLTYPE d3d12_command_list_WriteBufferImmediate(d3d12_comm
         }
         else
         {
-            FIXME_ONCE("VK_AMD_buffer_marker not supported by device.\n");
+            d3d12_command_list_end_current_render_pass(list, true);
```

Comment:
Is it a problem if this is spammed multiple times if count > 1?

Reply:
No, we'll just do nothing in that case.

Reply:
fwiw this already happens when calling any other method that has to end the current render pass.

Reply:
Figured, just wanted to make sure.